Makefile

```diff
@@ -37,11 +41,9 @@ ifdef STATIC
 	LDFLAGS=-a -ldflags '-extldflags \"-static\"'
 endif
 
-# Go tools
-GO      = go
```

Comment:
While this is best practice in C/C++ where different compilers are used, i think we only have one compiler in go right ?

So im fine with not having this as a parameter (i also see that other parts of this makefile directly reference go compiler directly)